src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
1) "p.sex === 'm'" and "p.sex === 'f" are duplicated in other functions.
Let's move these to separate functions, like:
isPersonMale(person) and isPersonFemale(person)

2) 'people.filter' are also duplicated, let's use only one filter() method, and modify the condition inside the filter, like this:

```suggestion
  const men = people.filter(person => isPersonMale(person) && (
    century
      ? Math.ceil(person.died / 100) === century
      : true
  ));
```